docs/v0.9/en/user-guide/operations/disaster-recovery/dr-solution-based-on-dual-active-standby.md

```diff
@@ -1 +1,48 @@
-# DR solution based on Dual Active-Standby
\ No newline at end of file
+# DR solution based on Dual Active-Standby
+
```

Comment:
Before anything else, it's important to note that this is a feature available only to enterprises.